include/aspect/newton.h

```diff
@@ -115,6 +126,7 @@ namespace aspect
        * explanation of the purpose of this factor.
        */
       double newton_derivative_scaling_factor;
+      std::pair<std::string,std::string> Newton_stabilisation;
```

Comment:
Keep this as two separate variables and document what they do. You can return them as a pair in `get_Newton_stabilization`, but set them as two separate input arguments in `set_Newton_stabilization`